x/gamm/types/msgs_test.go

```diff
@@ -34,97 +36,97 @@ func TestMsgSwapExactAmountIn(t *testing.T) {
 
 	tests := []struct {
 		name       string
-		msg        MsgSwapExactAmountIn
+		msg        gammtypes.MsgSwapExactAmountIn
 		expectPass bool
 	}{
 		{
 			name: "proper msg",
-			msg: createMsg(func(msg MsgSwapExactAmountIn) MsgSwapExactAmountIn {
+			msg: createMsg(func(msg gammtypes.MsgSwapExactAmountIn) gammtypes.MsgSwapExactAmountIn {
 				// Do nothing
 				return msg
 			}),
 			expectPass: true,
 		},
 		{
 			name: "invalid sender",
-			msg: createMsg(func(msg MsgSwapExactAmountIn) MsgSwapExactAmountIn {
+			msg: createMsg(func(msg gammtypes.MsgSwapExactAmountIn) gammtypes.MsgSwapExactAmountIn {
 				msg.Sender = invalidAddr.String()
 				return msg
 			}),
 			expectPass: false,
 		},
 		{
 			name: "empty routes",
-			msg: createMsg(func(msg MsgSwapExactAmountIn) MsgSwapExactAmountIn {
+			msg: createMsg(func(msg gammtypes.MsgSwapExactAmountIn) gammtypes.MsgSwapExactAmountIn {
 				msg.Routes = nil
 				return msg
 			}),
 			expectPass: false,
 		},
 		{
 			name: "empty routes2",
-			msg: createMsg(func(msg MsgSwapExactAmountIn) MsgSwapExactAmountIn {
-				msg.Routes = []SwapAmountInRoute{}
+			msg: createMsg(func(msg gammtypes.MsgSwapExactAmountIn) gammtypes.MsgSwapExactAmountIn {
+				msg.Routes = []gammtypes.SwapAmountInRoute{}
 				return msg
 			}),
 			expectPass: false,
 		},
 		{
 			name: "invalid denom",
-			msg: createMsg(func(msg MsgSwapExactAmountIn) MsgSwapExactAmountIn {
+			msg: createMsg(func(msg gammtypes.MsgSwapExactAmountIn) gammtypes.MsgSwapExactAmountIn {
 				msg.Routes[1].TokenOutDenom = "1"
 				return msg
 			}),
 			expectPass: false,
 		},
 		{
 			name: "invalid denom2",
-			msg: createMsg(func(msg MsgSwapExactAmountIn) MsgSwapExactAmountIn {
+			msg: createMsg(func(msg gammtypes.MsgSwapExactAmountIn) gammtypes.MsgSwapExactAmountIn {
 				msg.TokenIn.Denom = "1"
 				return msg
 			}),
 			expectPass: false,
 		},
 		{
 			name: "zero amount token",
-			msg: createMsg(func(msg MsgSwapExactAmountIn) MsgSwapExactAmountIn {
+			msg: createMsg(func(msg gammtypes.MsgSwapExactAmountIn) gammtypes.MsgSwapExactAmountIn {
 				msg.TokenIn.Amount = sdk.NewInt(0)
 				return msg
 			}),
 			expectPass: false,
 		},
 		{
 			name: "negative amount token",
-			msg: createMsg(func(msg MsgSwapExactAmountIn) MsgSwapExactAmountIn {
+			msg: createMsg(func(msg gammtypes.MsgSwapExactAmountIn) gammtypes.MsgSwapExactAmountIn {
 				msg.TokenIn.Amount = sdk.NewInt(-10)
 				return msg
 			}),
 			expectPass: false,
 		},
 		{
 			name: "zero amount criteria",
-			msg: createMsg(func(msg MsgSwapExactAmountIn) MsgSwapExactAmountIn {
+			msg: createMsg(func(msg gammtypes.MsgSwapExactAmountIn) gammtypes.MsgSwapExactAmountIn {
 				msg.TokenOutMinAmount = sdk.NewInt(0)
 				return msg
 			}),
 			expectPass: false,
 		},
 		{
 			name: "negative amount criteria",
-			msg: createMsg(func(msg MsgSwapExactAmountIn) MsgSwapExactAmountIn {
+			msg: createMsg(func(msg gammtypes.MsgSwapExactAmountIn) gammtypes.MsgSwapExactAmountIn {
```

Comment:
q: Would it make sense to split up pass-through changes into a separate PR so that we can backport it? 

Otherwise, backporting other work can get bottlenecked on this later

cc: @ValarDragon

Reply:
I agree with creating a new PR to change package name.

Reply:
I'm going to create it.

Reply:
ah thats a great point, should also be a really quick review + merge

Reply:
Thank you @vuong177 , merged https://github.com/osmosis-labs/osmosis/pull/2316